src/plone/restapi/__init__.py

```diff
@@ -1,3 +1,32 @@
 # -*- coding: utf-8 -*-
 from AccessControl import allow_module
 allow_module('json')
+
+from Products.Archetypes.ArchetypeTool import process_types, listTypes
+from Products.CMFCore import permissions
+
+import pkg_resources
+
+try:
```

Comment:
@buchi why can't we just use a testing.zcml that we only register for the test fixture here?